fastlane/lib/fastlane/helper/s3_client_helper.rb

```diff
@@ -50,12 +51,26 @@ def find_bucket!(bucket_name)
 
         return bucket
       end
-    end
 
-    private
+      private
 
-    def client
-      @client ||= Aws::S3::Client.new
+      def client
+        @client ||= Aws::S3::Client.new(
+          {
+            region: region,
+            credentials: create_credentials
+          }.compact
+        )
+      end
+
+      def create_credentials
+        return nil if access_key.nil? || @secret_access_key.nil?
```

Comment:
Is there reason why `secret_access_key` has the `@` in front of it and `access_key` doesn't? 🤔 Just asking so we can have consistency or a comment mentioning why 😇

Reply:
This is a small attempt at keeping the secret_access_key from being accessed using an attr_reader. I can make that private or if its not a concern I can just open it up alongside the access_key.

Reply:
I don't think its too much of a concern! I would go for consistency here if possible 😊